src/snowflake/snowpark/dataframe.py

```diff
@@ -664,14 +664,15 @@ def _execute_and_get_query_id(
         self, *, statement_params: Optional[Dict[str, str]] = None
     ) -> str:
         """This method is only used in stored procedures."""
-        return self._session._conn.get_result_query_id(
-            self._plan,
-            _statement_params=create_or_update_statement_params_with_query_tag(
-                statement_params or self._statement_params,
-                self._session.query_tag,
-                SKIP_LEVELS_THREE,
-            ),
-        )
+        with open_telemetry_context_manager(self._execute_and_get_query_id, self):
+            return self._session._conn.get_result_query_id(
+                self._plan,
```

Comment:
Does this capture the table sproc execution time?

Reply:
Yes, this is an action function so all time spend on this will be recorded